tools/build.sh

```diff
@@ -9,6 +9,7 @@
 
 set -e
 
+BUILD_RELEASE=True
```

Comment:
Maybe that's a bad variable name. It was just ment to trigger additional tests. I'll take a look!